index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw=="
+    crossorigin="anonymous" />
+    <link rel="stylesheet" href="/style.css">
+    <title>Midori | Home</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-items">
+            <div class="nav-brand">Midori</div>
+              <div class="input-icon icons-left">
+                <span class="material-icons">search</span> 
+                <input class="input" type="" id="" name="" placeholder="search"/> 
+              </div>
+              <div class="sub-menu"> 
+                <button class="btn btn-cta">Login</button>
+                <div class="header-icon">
+                   <span class="material-icons">shopping_cart</span>                   
+                </div>
+                <div class="header-icon"> 
+                    <span class="material-icons">favorite_border</span>                   
+                </div>                         
+             </div> 
+        </div>
+    </header>    
+    <section class="main-content-page">      
+        <div class="three-box-section"> 
+           <article class="category">
+             <img src="/images/Plant1.jpg" alt="plant" class="img card-image">
+             <p class="category-text-overlay text-bold">Indoor Plants</p>
+           </article>
+
+           <article class="category">     
+             <img src="/images/Plant2.jpg" alt="plant" class="img card-image">
+             <p class="category-text-overlay text-bold">Flowering Plants</p>        
+           </article>
+
+           <article class="category"> 
+             <img src="/images/Plant3.jpg" alt="plant" class="img card-image">
+             <p class="category-text-overlay text-bold">Outdoor Plants</p> 
+           </article>   
+        </div>
+
+        <div class="hero-img">
+             <img src="/images/Plant4.jpg" alt="hero-img" >
+             <div class="hero-img-overlay">
+               <h2>Indoor Plants</h2>
+               <h4>Live with Nature</h4>
+               <button class="btn btn-secondary">Shop Now</button>
+             </div>
+        </div>
+
+        <div class="two-box-section"> 
+           <article class="sub-section-container"> 
+             <div class="sub-section">
+               <div class="sub-section-image">
+                 <img src="/images/Plant1.jpg" alt="" class="ss-image">
+               </div>
```

Comment:
I liked the fact that you are using consistent color naming convention for css but here i see some alt tags values missing. Rest all looks good.